openpype/plugins/publish/extract_review_slate.py

```diff
@@ -267,23 +336,24 @@ def process(self, instance):
                 "-f", "concat",
                 "-safe", "0",
                 "-i", conc_text_path,
-                "-c", "copy",
+                "-c:v", "copy",
+                output_path
             ]
-            # NOTE: Added because of OP Atom demuxers
```

Comment:
It is required to add format definitions into contatenation otherwise mxf demuxer will use default mxf format (OP1a) to output.

Reply:
@jrsndl Remainding: With removing of these lines the output won't be OP Atom but OPa1. Concatenation also requires to specify output format otherwise default format option is used..

Reply:
Indeed. I had this already fixed. Oh well, Git challenged...